contracts/common/EtherPaymentFallback.sol

```diff
@@ -1,12 +1,17 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity >=0.7.0 <0.9.0;
 
-/// @title EtherPaymentFallback - A contract that has a fallback to accept ether payments
-/// @author Richard Meissner - <richard@gnosis.pm>
+/**
+ * @title EtherPaymentFallback - A contract that has a fallback to accept ether payments.
+ * @author Richard Meissner - @rmeissner
+ */
 contract EtherPaymentFallback {
     event SafeReceived(address indexed sender, uint256 value);
 
-    /// @dev Fallback function accepts Ether transactions.
+    /**
+     * @notice Receive function accepts Ether transactions.
```

Comment:
Need we state the function name, or should we keep "Fallback"? If so, I wouldn't capitalise it.

Reply:
I think it's better to do so because `fallback` and `receive` are two distinct functions in Solidity